ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -489,6 +537,21 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
+    private func makeBottomBarLayoutGuideConstraints(with bottomBarView: UIView) -> [NSLayoutConstraint] {
+        let requiredConstraints = [
+            layoutGuide.leadingAnchor.constraint(equalTo: bottomBarView.leadingAnchor),
+            layoutGuide.trailingAnchor.constraint(equalTo: bottomBarView.trailingAnchor),
+            layoutGuide.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            layoutGuide.topAnchor.constraint(lessThanOrEqualTo: view.bottomAnchor)
+        ]
+
+        // bottomBarView will go off-screen when it's hidden, so this constraint is not always required.
+        let breakableConstraint = layoutGuide.topAnchor.constraint(equalTo: bottomBarView.topAnchor)
```

Comment:
can we be more specific than "breakableConstraint". Breakable says something about its priority, but doesn't say which constraint it is